host/tests/rmfAudioClasses/rmfAudio.py

```diff
@@ -0,0 +1,568 @@
+                    "query": "Select the audio capture type :",
+                    "input": str(capture_type)
+                }
+        ]
+        result = self.utMenu.select(self.testSuite, "Close RMF Audio Capture Handle", promptWithAnswers)
+
+    def updateSettings(self, capture_type:int=1, settings_update:int=0, capture_format:int=1, sampling_rate:int=5, fifo_size:int=65536, threshold:int=8192):
+        """
+        Update RMF_AudioCapture_Settings values if required.
+
+        Args:
+            capture_type (int, optional): 1 for primary data capture (default), 2 for auxiliary data capture.
+            settings_update (int, optional): Choice whether default settings needs to be updated (default is 0).
+            capture_format (int, optional): RMF Audio Capture Format for audio data (default is racFormat_e16BitStereo i.e 1).
+            sampling_rate (int, optional): RMF Audio Capture Sampling Rate (default is racFreq_e48000 i.e 5).
+            fifo_size (int, optional): FIFO size in bytes (default is 65k, 65536)
+            threshold (int, optional): FIFO data callback threshold in bytes (default is 8k, 8192)
+
+        Returns:
+            None
+        """
+        promptWithAnswers = [
+                {
+                    "query_type": "direct",
+                    "query": "Select the audio capture type :",
+                    "input": str(capture_type)
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Do you want to update default settings ? (0 for No, 1 for Yes)",
+                    "input": str(settings_update)
+                }
+        ]
+
+        # If the user chooses update settings, prompt for settings update
+        if settings_update == 1:
+            promptWithAnswers.append(
+                {
+                    "query_type": "direct",
+                    "query": "Select the capture format to update, use -1 to retain default value :",
+                    "input": str(capture_format)
+                })
+            promptWithAnswers.append(
+                {
+                    "query_type": "direct",
+                    "query": "Select the Sampling Rate, use -1 to retain default value :",
+                    "input": str(sampling_rate)
+                })
+            promptWithAnswers.append(
+                {
+                    "query_type": "direct",
+                    "query": "Enter FIFO size in bytes, use -1 to retain default value :",
+                    "input": str(fifo_size)
+                })
+            promptWithAnswers.append(
+                {
+                    "query_type": "direct",
+                    "query": "Enter data callback threshold in bytes, used to check jitter (max 1/4th of FIFO), use -1 to retain default value :",
+                    "input": str(threshold)
+                })
+
+        result = self.utMenu.select(self.testSuite, "Update settings", promptWithAnswers)

```

Comment:
please update the 
result = self.utMenu.select(self.testSuite, "Update settings", promptWithAnswers) with 

`result = self.utMenu.select(self.testSuite, "Get and update default settings", promptWithAnswers)`

Reply:
updated as suggested